apps/tai/lib/tai/markets/order_book.ex

```diff
@@ -144,4 +168,8 @@ defmodule Tai.Markets.OrderBook do
       end
     )
   end
+
+  defp broadcast_change_set? do
```

Comment:
I feel like this should be passed into `start_link` and be stored on the state. Can you also add it to the global config https://github.com/fremantle-capital/tai/blob/master/apps/tai/lib/tai/config.ex#L35

Reply:
I've added this to global config. I'm not too happy with the name spacing for the variable. Also, as a side effect, this gives you control over individual venues.